spec/unit/configurer_spec.rb

```diff
@@ -1037,6 +1058,9 @@ def expects_neither_new_or_cached_catalog
     end
 
     it "should not make multiple node requets when the server is found" do
+      response = Net::HTTPOK.new(nil, 200, 'OK')
```

Comment:
Could we get other 2xx responses from the server?  This is the simple status endpoint, so also probably OK.

Reply:
Per https://puppet.com/docs/puppetserver/5.0/status-api/v1/simple.html#response it only returns 200 or 503. I assume a generic 500 is also possible if it's in a bad state.